packages/ui-components/.gitignore

```diff
@@ -1,6 +1,6 @@
 dist/
 www/
-loader/
+./loader/
```

Comment:
why the change to a relative path?

Reply:
Stenciljs generate a loader directory, so I assume that change is to git get the changes on the kv-loader. Probably loader isn't a good name, since stenciljs use it.

Reply:
@ricardoarasilva remove the `.` at the beginning

Reply:
@ricardoarasilva after make the change you need to mark as resolved